starter-core/src/main/java/io/micronaut/starter/feature/function/awslambda/template/awsLambdaFunctionRequestHandlerGroovyJunit.rocker.raw

```diff
@@ -0,0 +1,44 @@
+@import io.micronaut.starter.application.Project
+
+@args (
+Project project
+)
+
+@if (project.getPackageName() != null) {
+package @project.getPackageName()
+}
+
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+
+import static org.junit.jupiter.api.Assertions.assertEquals
+
+class FunctionRequestHandlerTest {
+
+    private static FunctionRequestHandler handler
+
+    @@BeforeAll
+    static void setupServer() {
+        handler = new FunctionRequestHandler()
+    }
+
+    @@AfterAll
+    static void stopServer() {
+        if (handler != null) {
+            handler.getApplicationContext().close()
+        }
+    }
+
+    @@Test
+    void testHandler() {
+        APIGatewayProxyRequestEvent request = new APIGatewayProxyRequestEvent()
+        request.setHttpMethod("GET")
+        request.setPath("/")
```

Comment:
nit: could be:

```
        request.httpMethod = "GET"
        request.path = "/"
```

Reply:
resolved via https://github.com/micronaut-projects/micronaut-starter/pull/1209/commits/851470e7ee8747c5aef117b3558762162ff73a96